src/posix.mak

```diff
@@ -532,8 +532,8 @@ $G/dmd.conf: $(SRC_MAKE)
 ######## optabgen generates some source
 optabgen_output = debtab.d optab.d cdxxx.d elxxx.d fltables.d tytab.d
 
-$G/optabgen: $C/optabgen.c $C/cc.h $C/oper.h
-	$(HOST_CXX) $(CXXFLAGS) -I$(TK) $< -o $G/optabgen
+$G/optabgen: $(HOST_DMD) $C/optabgen.d $C/cc.d $C/oper.d
```

Comment:
Depend on `$(HOST_DMD_PATH)` and put the dependency at the end. Otherwise $< won't refer to optabgen.d